examples/java/greetings_project/WORKSPACE

```diff
@@ -0,0 +1,25 @@
+load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
+
+jvm_maven_import_external(
+    name = "junit",
+    artifact = "junit:junit:4.12",
```

Comment:
Can we use the newest version (5.9.1)? Or at least 4.13.2, this one contains CVE

Reply:
I updated this and removed the other external libraries.